staging/src/k8s.io/component-base/config/types.go

```diff
@@ -62,6 +62,12 @@ type LeaderElectionConfiguration struct {
 	// resourceLock indicates the resource object type that will be used to lock
 	// during leader election cycles.
 	ResourceLock string
+	// resourceName indicates the name of resource object that will be used to lock
+	// during leader election cycles.
+	ResourceName string
+	// resourceName indicates the namespace of resource object that will be used to lock
+	// during leader election cycles.
+	ResourceNamespace string
```

Comment:
is resource tenant not required? does it assuming implicitly "system" tenant?

Reply:
This struct is used for scheduler and KCM leader election only. Not related to multi-tenancy